packages/analytics-js-plugins/package.json

```diff
@@ -73,6 +73,7 @@
     "@preact/signals-core": "1.4.0",
     "@rudderstack/analytics-js-common": "3.0.0-beta.9",
     "crypto-js": "4.1.1",
```

Comment:
can we now remove this one?

Reply:
This package is still used in the v1.1 package. We can remove it once we upgrade that too later.

Reply:
but this is in the package json of the plugins that is not relevant to v1.1